### PR TITLE
add `cellranger vdj` test data

### PR DESCRIPTION
**Pull request by klkeys**
added FASTQs to test `cellranger vdj` for immune profiling

the original data are from the 10X Genomics tutorial for immune profiling (see [here](https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/tutorial/tutorial-vdj))

data here are subsampled from the original FASTQs

included a [README.md](data/genomics/homo_sapiens/illumina/10xgenomics/cellranger_vdj/README.md) with details

### CODE REVIEW

**Review by SPPearce — APPROVED** [review]

**Comment by SPPearce** [comment]
Do you need two lanes of this data? 
You'll need R1 and R2, but do you need both lanes, or would the tools work with just one lane? That'd still give you your 10000 reads you need.

**Comment by klkeys** [comment]
"need" is a strong word 😅 in my mind the test as arranged (successfully) checks for multilane samples which are an important real-world phenomenon with `cellranger` runs

that caveat aside, I will arrange a single lane test instead 👍🏼

**Comment by klkeys** [comment]
`cellranger vdj` test data now uses only 1 lane of data

File: data/genomics/homo_sapiens/illumina/10xgenomics/cellranger_vdj/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+├── README.md
+├── refdata-cellranger-vdj-GRCh38-alts-ensembl-5.0.0
+│   ├── fasta
+│   │   ├── regions.fa
+│   │   └── supp_regions.fa
+│   └── reference.json
+└── subsampled_sc5p_v2_hs_B_1k_b_fastqs
+    ├── subsampled_sc5p_v2_hs_B_1k_b_S1_L001_R1_001.fastq.gz
+    ├── subsampled_sc5p_v2_hs_B_1k_b_S1_L001_R2_001.fastq.gz
+    ├── subsampled_sc5p_v2_hs_B_1k_b_S1_L002_R1_001.fastq.gz
```
Comment (SPPearce) [inline data/genomics/homo_sapiens/illumina/10xgenomics/cellranger_vdj/README.md:62]:
```suggestion

```

File: data/genomics/homo_sapiens/illumina/10xgenomics/cellranger_vdj/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    └── subsampled_sc5p_v2_hs_B_1k_b_S1_L002_R2_001.fastq.gz
```
Comment (SPPearce) [inline data/genomics/homo_sapiens/illumina/10xgenomics/cellranger_vdj/README.md:63]:
```suggestion

```

**Review by SPPearce — APPROVED** [review]
It just halves the amount of data being added to an already very large repository.
If you need the other lanes for testing a pipeline, that can be put into a pipeline repo of test-data.

File: data/genomics/homo_sapiens/illumina/10xgenomics/cellranger_vdj/README.md (hunk 2 above)
Comment (klkeys) [reply data/genomics/homo_sapiens/illumina/10xgenomics/cellranger_vdj/README.md:62]: good eyes, thank you 🙏🏼

**Pull request merged**